From 8c11cc20507f361d0b82abd12f183aab36c859b9 Mon Sep 17 00:00:00 2001
From: ruslanbay <67730802+ruslanbay@users.noreply.github.com>
Date: Thu, 27 Nov 2025 22:38:11 +0100
Subject: [PATCH 06/16] export find_vm_area to resolve the following error:
 modpost: "find_vm_area" [drivers/media/pci/intel/ipu4/intel-ipu4.ko]
 undefined!

---
 mm/vmalloc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index ebeb6b2e1..f7d5b3161 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -2588,6 +2588,7 @@ struct vm_struct *find_vm_area(const void *addr)
 
 	return va->vm;
 }
+EXPORT_SYMBOL(find_vm_area);
 
 /**
  * remove_vm_area - find and remove a continuous kernel virtual area
-- 
2.34.1

